src/routes/results/[results_string]/+page.svelte

```diff
@@ -97,6 +100,17 @@
 </div>
 
 <style>
+	a {
+		color: var(--charcoal);
+		font-weight: 600;
+		text-decoration: none;
+		font-style: normal;
+	}
+	a:hover,
+	a:active,
+	a:focus {
+		font-style: italic;
```

Comment:
nit: wdyt of making bolding the font instead of italicizing? I think this is a bit more in line with other styling we have on the results page

Reply:
sounds good! I had pulled the bolded/italic idea from the nav bar on https://www.allwecansave.earth/, but I think the non-bolded start looks better. just updated.